react/features/base/media/components/native/VideoTransform.js

```diff
@@ -167,6 +173,13 @@ class VideoTransform extends Component<Props, State> {
      * @inheritdoc
      */
     componentWillMount() {
+        const { height, width } = Dimensions.get('window');
```

Comment:
Note that the window dimensions might not be the app dimensions. See https://github.com/jitsi/jitsi-meet/blob/master/react/features/base/responsive-ui/components/DimensionsDetector.native.js for dynamically detecting dimensions.

Reply:
In which cases can window dimensions be different from the app screen size?

Reply:
Probably if you double press home button on iOS which bring the app half screen. I don't know that for sure, but there are also some split window modes on tablets or something like that.

Reply:
split view, for insatance

Reply:
Ok, added a more flexible solution :)

Reply:
So this gets the dimensions on componentDidMount, but resize will probably not unmount component and the value will be outdated. The docs[1] advise to call it on every render or maybe this DimensionDetector mentioned by @saghul will work, but you need to check how it works as I don't know that...
[1]: https://facebook.github.io/react-native/docs/dimensions.html#get